src/util/__mocks__/urlparams.js

```diff
@@ -0,0 +1,13 @@
+import wa_9 from "../../test/fixtures/districts/wa_9.json";
+import districts from "../../test/fixtures/districts/districts.json";
+
+export default function getUrlParameter(params, name) {
```

Comment:
instead of this, write tests for getUrlParameter to confirm that it is working as desired.